docs/runtime/shell.md

```diff
@@ -149,7 +152,7 @@ The following JavaScript objects are supported for redirection to:
 To redirect the output from JavaScript objects to stdin, use the `<` operator:
 
 ```js
-import { $, file } from "bun";
+import { $ } from "bun";
```

Comment:
That redundant import confused me. I thought we cat a file or something until I realized it may be a leftover